### PR TITLE
Add dataset name param

### PR DESCRIPTION
**Pull request by bahlo**
This adds a dataset name parameter and bump version to 0.4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,12 +33,12 @@ Add the following to your Cargo.toml:
 tracing-axiom = "0.3"
```
Comment (bdsqqq) [inline README.md]: Should this be 0.4?

File: README.md [hunk 2]
```diff
@@ -33,12 +33,12 @@ Add the following to your Cargo.toml:
...
+Create a dataset in Axiom and export the name as `AXIOM_DATASET`.
+Then create an API token with ingest permission into that dataset in
+[the Axiom settings](https://cloud.axiom.co/settings/profile) and export it as
+`AXIOM_TOKEN`.
 
-Otherwise create an API token in [the Axiom settings](https://cloud.axiom.co/settings/profile) and export it as `AXIOM_TOKEN`. 
-The token needs to have ingest permission to all datasets, using it will automatically create a `_traces` dataset for you.
-
-Set up tracing in one line like this:
+Now you can set up tracing in one line like this:
```
Comment (bdsqqq) [inline README.md:41]: Very nice, this keeps the tone "this, then, then, now" better 👍👍

**Review by bdsqqq — COMMENTED** [review]
👍👍

**Review by lukasmalkmus — APPROVED** [review]
Please address outstanding conversations, otherwise LGTM.

File: README.md (hunk 1 above)
Comment (bahlo) [reply README.md]: Oops, yep

**Pull request merged**